generators/app/templates/README.md

```diff
@@ -2,16 +2,50 @@
 
 <%= projectDescription %>
 
+You can find the objective of this project and dependencies in the [ARCHITECTURE.md](ARCHITECTURE.md) file within this repository.
+
+> **Reminder:** [Making READMEs readable](https://pages.18f.gov/open-source-guide/making-readmes-readable/).
+
+## Folder Structure
+
+```
+.
+├── .github
+|   └── PULL_REQUEST_TEMPLATE.md
+├── bin
+├── config
+├── lib
+├── src
+|   ├── assets
+|   ├── models
+|   └── services
+└── tests
+```
+
+:sparkles: **.github**: This is our integration with github service, we highlight that we are going to provide a [Pull Request Template](https://help.github.com/articles/creating-a-pull-request-template-for-your-repository/).
+
+:information_source: **bin**: Find the wrappers to start the services we provide within this code here, `package.json` is going to have commands pointing to those files.
+
+:memo: **config**: Environment variables or `.env` file? This is the place to answer these type of questions.
```

Comment:
Suggestion:

This folder contains javascript files that are responsible to read the `.env` file, constructing and exporting a module to our app, that can be used through `const configs = require("getconfig")`